packages/grid/x-data-grid/src/hooks/features/clipboard/useGridClipboard.ts

```diff
@@ -81,16 +84,36 @@ export const useGridClipboard = (apiRef: React.MutableRefObject<GridPrivateApiCo
         return;
       }
 
-      apiRef.current.unstable_copySelectedRowsToClipboard();
+      let textToCopy = '';
+      const selectedRows = apiRef.current.getSelectedRows();
+      if (selectedRows.size > 0) {
+        textToCopy = apiRef.current.getDataAsCsv({
+          includeHeaders: false,
+          // TODO: make it configurable
+          delimiter: '\t',
```

Comment:
Now that we make import delimiter configurable using `unstable_splitClipboardText`, I feel it could be useful in some use-cases like copying data to and from a specific spreadsheet application that is configured with a specific delimiter. 🤔 
What do you think?

Reply:
Sure 👍
Added a `clipboardCopyCellDelimiter` prop to avoid complex changes and keep using `getDataAsCsv` method here.
Also, added a demo with comma cell delimiter in https://deploy-preview-7389--material-ui-x.netlify.app/x/react-data-grid/clipboard/#format-of-the-pasted-data